src/App.jsx

```diff
@@ -14,33 +15,91 @@
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [sortField, setSortField] = useState('');
+  const [reversed, setReversed] = useState('false');
+  let visibleGoods = [...goodsFromServer].sort((good1, good2) => {
+    switch (sortField) {
+      case 'sortAlphabetically':
+        return good1[0].localeCompare(good2[0]);
+      case 'sortLength':
+        if (reversed === 'true') {
+          return good2.length - good1.length;
+        }
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+        return good1.length - good2.length;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+      default:
+        return 0;
+    }
+  });
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (!reversed) {
+    visibleGoods = visibleGoods.reverse();
+  }
+
+  function Reset() {
+    setSortField('');
+
+    setReversed('false');
```

Comment:
The `setReversed('false')` call sets `reversed` to a string instead of a boolean. Use `setReversed(false)` to ensure `reversed` remains a boolean.